api/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
-
+from django.utils import timezone
```

Comment:
Removed the last line in my comment. I understood from Jure's comment on project that you do not need to set default=timezone(now) for DateTimeField.